src/main/java/org/jboss/netty/handler/codec/http/cookie/CookieUtil.java

```diff
@@ -134,7 +134,21 @@ static int firstInvalidCookieValueOctet(CharSequence cs) {
     }
 
     static int firstWarnCookieValueOctet(CharSequence cs) {
-        return firstInvalidOctet(cs, WARN_COOKIE_OCTETS);
+
+        // This is a list of characters that are not generally allowed in cookies per spec.
+        BitSet bits = WARN_COOKIE_OCTETS;
+
+        // If you find a character in the list...
+        for (int i = 0; i < cs.length(); i++) {
```

Comment:
no null safety here - is this ok?

Reply:
The library does similar things without null checks.  The allocation happens once in a static context as the JVM is starting up.  There's no reasonable runtime remedy if is actually null.  I'm willing to let this one slide.